src/dom_types.rs

```diff
@@ -1374,4 +1374,28 @@ pub mod tests {
 
         assert_eq!(expected, get_node_html(&node));
     }
+
+    /// Test that `style!` macro accpet types that have to_css_value() function
```

Comment:
accept (typo)
`to_css_value` (add backticks pls)

Reply:
>  to_css_value (add backticks pls)

Sorry I don't understand what you mean by this. Can you elaborate.

Reply:
Sure.

Original: 
```rust
/// Test that `style!` macro accpet types that have to_css_value() function
```
New: 
```rust
/// Test that `style!` macro accept types that have `to_css_value` function
```
`to_css_value` is marked as code by backticks (`).
 - To make it more readable.
 - Pedantic clippy is often clever enough to warn you if you forget to do it. (However it's not clever enough to recognize all code fragments in comments at the moment.)

Ideally we should use also links in comments - see https://stackoverflow.com/questions/31582064/how-to-link-to-other-fns-structs-enums-traits-in-rustdoc.

Reply:
Done.

Reply:
> Ideally we should use also links in comments - see https://stackoverflow.com/questions/31582064/how-to-link-to-other-fns-structs-enums-traits-in-rustdoc.

Wait, where should I link this method, we have it in 3 traits :D.

I don't think linking `to_css_value` is needed here since
1- we have 3 places where this method defined.
2- we already hide it's docs since we don't want users to know about `style!` internal API.

Reply:
> I don't think linking to_css_value is needed here since

I agree. It was rather a general tip than requirement.

Reply:
That was helpful tip, that I didn't know it was so simple to do!.